src/api/channel/channel.ts

```diff
@@ -0,0 +1,157 @@
+type ChannelId = string;
+
+declare interface Listener{    
+    /**
+     * Remove this event listener
+     * 
+     */
+    removeEventListener(): void;
+}
+
+/**
+ * Object representing a context channel.
+ * 
+ * All interactions with a context channel happen through the methods on here.
+ */
+declare class Channel {
+    /**
+     * Constant that uniquely identifies this channel. Will be generated by the service, and guarenteed to be unique
+     * within the set of channels registered with the service.
+     * 
+     * In the case of `desktop` channels (see {@link DesktopChannel}), these id's _should_ persist across sessions. The 
+     * channel list is defined by the service, but can be overridden by a desktop owner. If the desktop owner keeps 
```

Comment:
This is introducing a new concept "desktop owner", which isn't defined in our spec:

https://fdc3.finos.org/docs/1.0/api/api-spec

Any new terminology should be clearly defined in the documentation!